cluster/src/test/java/com/hartwig/pipeline/execution/vm/unix/RedirectStdoutCommandTest.java

```diff
@@ -0,0 +1,20 @@
+package com.hartwig.pipeline.execution.vm.unix;
+
+import static java.lang.String.format;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.hartwig.pipeline.calling.command.SamtoolsCommand;
+
+import org.junit.Test;
+
+public class RedirectStdoutCommandTest {
+    @Test
+    public void createsValidCommand() {
+        String grepString = "view file.bam";
+        String outputFile = "/some/output/file";
+        assertThat(new RedirectStdoutCommand(new SamtoolsCommand(grepString), outputFile).asBash()).isEqualTo(format("(/opt/tools/samtools/1.14/samtools %s 1> %s)",
```

Comment:
I do find this a bit confusing, and it introduces a dependency on `SamtoolsCommand`. Could we simplify it to just test the functionality the class under test provides?

Reply:
Agreed and will replace with mock of BashCommand instead.